scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py

```diff
@@ -379,6 +381,19 @@ def config_lvm(self):
         for device in metadata_devices:
             create_lvm(self, dev_count, device)
             dev_count += 1
+            lv_md_name = f"lv_raw_md{dev_count}"
+            cmd = f"lvs -o lv_path | grep {lv_md_name}"
+            res = execute_command(self, cmd)
+            lv_path = res[0].rstrip("\n")
+            cmd = f"lvs {lv_path} -o LV_SIZE --noheadings --units b --nosuffix"
+            res = execute_command(self, cmd)
+            lv_size = res[0].rstrip("\n")
+            if lvm_min_size is None:
+                lvm_min_size = lv_size
+            lvm_min_size = min(lv_size, lvm_min_size)
+    if lvm_min_size: 	    
+        cmd = f'sed -i "s/^\(MOTR_M0D_IOS_BESEG_SIZE=*\).*/\1{lvm_min_size}/" {motr_config_file}'
```

Comment:
Need to come up with a right utility to update /etc/sysconfig/motr, mistakes here can cause unintended consequences. Also need to log pre and post changes to config file.

Reply:
will create a different PR for updating /etc/sysconfig/motr file using python